### PR TITLE
Hidden labels

### PR DESCRIPTION
**Pull request by KellyVance**
Addition of hidden labels, addition of alternative labels, and derivation of additional UOMs (magnitude additions to existing types) to align vocabularies with Petroleum data submission templates.

integration and technical changes, no conceptual additions or changes to vocabulary structure.

### CODE REVIEW

**Review by johnmckellar — APPROVED** [review]
All look good to me.

File: vocabularies/geounits.ttl [hunk 1]
```diff
@@ -546,6 +555,7 @@ geou:MegaBBLOE a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/geou> .
 
 geou:MegaBOE a skos:Concept ;
+    skos:altLabel "Million BBLs Oil Equiv. (MMBOE)"@en,
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geounits.ttl:558]:
```suggestion
    skos:altLabel "Million BBLs Oil Equiv. (MMBOE)"@en ;
```

File: vocabularies/geounits.ttl [hunk 2]
```diff
@@ -601,12 +611,24 @@ geou:MegaM3 a skos:Concept ;
...
+    rdfs:isDefinedBy geou: ;
+    skos:definition "A measure of mass equivalent to one thousand tonnes."@en ;
+    skos:notation "kt" ;
+    skos:inScheme <http://linked.data.gov.au/def/geou> ;
+    skos:prefLabel "Kilotonnes"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/geou> .
+
 geou:MegaTON_M a skos:Concept ;
+    skos:altLabel "Megatonnes (Mt)"@en,
+        "Million Tonnes"@en
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geounits.ttl:626]:
```suggestion
        "Million Tonnes"@en  ;
```

**Review by DavidCrosswellGSQ — CHANGES_REQUESTED** [review]
A couple of changes required to geounits

File: vocabularies/geounits.ttl (hunk 1 above)
Comment (johnmckellar) [reply vocabularies/geounits.ttl:558]: OK

File: vocabularies/geounits.ttl (hunk 2 above)
Comment (johnmckellar) [reply vocabularies/geounits.ttl:626]: OK too

Comment (johnmckellar) [reply vocabularies/geounits.ttl:626]: Sorry, missed this & OK

**Comment by KellyVance** [comment]
geounits syntax errors fixed.

**Review by DavidCrosswellGSQ — APPROVED** [review]

**Pull request merged**